src/lsp/main.ts

```diff
@@ -755,8 +780,10 @@ async function getDocumentSymbols(
     return result;
 }
 
-async function getServerInfo(lspClient: LanguageClient): Promise<any> {
-    return lspClient.sendRequest('clojure/serverInfo/raw');
+function getServerInfo(lspClient: LanguageClient): Promise<any> {
+    return lspClient.sendRequest('clojure/serverInfo/raw').catch((e) => {
+        console.error('clojure/serverInfo/raw failed:', e);
+    });
```

Comment:
This might return a Promise that resolves to `undefined`, so we might get an error from code like the below which expects the `getServerInfo` promise to be rejected if the `serverInfo` request fails.

So with the code before, the code below the `await` wouldn't have been run if the `serverInfo` request fails, but with the `catch` above, it will be run (since the promise was resolved instead of rejected), but `serverInfo` would be `undefined`.

```clojure
async function openLogFile(): Promise<void> {
  const client = getStateValue(LSP_CLIENT_KEY);
  if (client) {
    const serverInfo = await getServerInfo(client);
    const logPath = serverInfo['log-path'];
    void vscode.window.showTextDocument(vscode.Uri.file(logPath));
  } else {
    void vscode.window.showInformationMessage(SERVER_NOT_RUNNING_OR_INITIALIZED_MESSAGE);
  }
}
```

I'm not positive about this though. Maybe @corasaurus-hex can verify.